Sources/SwiftDocC/Semantics/Symbol/DocumentationDataVariants.swift

```diff
@@ -104,9 +104,21 @@ extension DocumentationDataVariants {
     }
     
     /// Convenience API to access the first variant, or the default value if there are no registered variants.
+    ///
+    /// > Important:
```

Comment:
Nit: Should we "soft" deprecate this with a deprecation directive?

Reply:
I'm not sure what you're referring to here. A `@DeprecationSummary` directive would need to go in a documentation extension file, and I don't feel that that would make this information more visible.

Reply:
Ah right- good point. I forgot DocC doesn't support directives in doc comments... Feels like a bug for `DeprecationSummary` to be honest.

I've seen other writers use that directive to deprecate something in the docs without causing a warning so it seemed appropriate here, but definitely not worth adding a doc extension.